### DISCUSSION TITLE
Mi primera clase

### DISCUSSION THREAD

**Discussion by frankYanez** [Launch-X-Latam/MisionFrontEnd#487] [Team Posts]
Hola buenas tardes. Disculpa l molestia, es la primera vez que veo todo esto. Me llego el correo para iniciar la formacion backend, lamentablemente por motivos de fuerza mayor no pude hacer el front-end. Queria saber si puedo realizar la parte del backend ain problemas (ya tengo conociemientos en front-en) o a quien debo comunicarme para aliatarme para el back. Mil gracias

**Comment by jko16** [comment]
Hola, Tienes hasta el 18 de noviembre para igualarte la parte frontend

**Comment by frankYanez** [comment]
Perfecto. Pero no puedo acceder a las clases! No tengo ningun equipo tampoco

**Comment by MrAlizarinCrimson** [comment]
Buen día me pasa igual que Francisco Yáñez
